### PR TITLE
Create and populate `plasmapy.formulary.frequencies`

### PR DESCRIPTION
**Pull request by rocco8773**
This PR moves frequency plasma parameters from `plasmapy.formulary.parameters` to `plasmapy.formulary.frequencies` in accordance with issue #1433.

### CODE REVIEW

**Comment by rocco8773** [comment]
@namurphy The changelog check is failing here because I fixed a typo in the changelog for PR #1434.  If you want to add this fix to your PR #1415, then I'm ok with that and can revert the change for this PR.  I'm fine doing it here or there.

File: plasmapy/formulary/parameters.py [hunk 1]
```diff
@@ -73,74 +56,45 @@
...
+for modname, name in funcs_to_deprecate_wrap:
+    globals()[name] = deprecated(
+        since="0.7.0",
+        warning_type=PlasmaPyFutureWarning,
+        message=(
+            f"The {name}() function has been moved to "
+            f"plasmapy.formulary.{modname}.  Update your import to get "
+            f"rid of this warning."
+        ),
+    )(getattr(globals()[f"{modname}"], name))
```
Comment (namurphy) [inline plasmapy/formulary/parameters.py:95]: Nice!

**Review by namurphy — APPROVED** [review]
LGTM!

**Comment by namurphy** [comment]
> @namurphy The changelog check is failing here because I fixed a typo in the changelog for PR #1434. If you want to add this fix to your PR #1415, then I'm ok with that and can revert the change for this PR. I'm fine doing it here or there.

It seems sensible to fix this typo here.  If we (temporarily) add the "no changelog entry needed" label then we can skip the changelog checks.

**Pull request merged**